gnovm/pkg/gnolang/gonative.go

```diff
@@ -685,22 +456,18 @@ func go2GnoValue2(alloc *Allocator, store Store, rv reflect.Value, recursive boo
 	case reflect.Chan:
 		panic("not yet implemented")
 	case reflect.Func:
-		// NOTE: the type may be a full gno type, either a
-		// *FuncType or *DeclaredType.  The value may still be a
-		// *NativeValue though, and the function can be called
-		// regardless.
-		tv.V = alloc.NewNative(rv)
+		panic("not yet implemented")
 	case reflect.Interface:
 		// special case for errors, which are very often used especially in
 		// native bindings
 		if rv.Type() == tError {
 			tv.T = gErrorType
-			if !rv.IsNil() {
-				tv.V = alloc.NewNative(rv.Elem())
-			}
+			//if !rv.IsNil() {
+			//	//tv.V = NewErrorInterfaceValue(alloc, rv)
+			//}
 			return
 		}
```

Comment:
Here @thehowl should we create an specific interface value only for error interfaces ?
You said about the InterfaceValue type
> No, it doesn't exist because they are passed around simply as concrete values. I don't think we need to create it either

but I do not understand how, sorry

Reply:
thanks 🚀 ,
I got confused because we do have some functions that returns error interface 
So I got some panics....
but I removed it and it is working 79c6d1c08c417207874e69f549a049dd212dc6c9